.jenkins/pace_physics_cache_setup.sh

```diff
@@ -7,5 +7,9 @@ export LONG_EXECUTION=1
 .jenkins/jenkins.sh run_physics_regression_tests ${backend} ${experiment}
 cd buildenv && git reset --hard && cd ..
 .jenkins/jenkins.sh run_physics_parallel_regression_tests ${backend} ${experiment}
+cd buildenv && git reset --hard && cd ..
```

Comment:
FYI `git` has an option `-C` which uses the following argument as the directory, so this line could be
```
git -C buildenv reset --hard HEAD
```